server/plugin/webhook.go

```diff
@@ -1019,18 +1033,42 @@ func (p *Plugin) handleCommentAssigneeNotification(event *github.IssueCommentEve
 	if len(splitURL) < 2 {
 		return
 	}
+
+	eventType := splitURL[len(splitURL)-2]
 	var templateName string
-	switch splitURL[len(splitURL)-2] {
+	switch eventType {
 	case "pull":
 		templateName = "commentAssigneePullRequestNotification"
 	case "issues":
 		templateName = "commentAssigneeIssueNotification"
 	default:
-		p.API.LogWarn("Unhandled issue type", "type", splitURL[len(splitURL)-2])
+		p.API.LogWarn("Unhandled issue type", "Type", eventType)
```

Comment:
While we are here...
```suggestion
		p.API.LogDebug("Unhandled issue type", "Type", eventType)
```